packages/core/src/components/way-select/way-select.scss

```diff
@@ -0,0 +1,101 @@
+.w2w-select {
+  $height: 40px;
+  $color-selected: rgba(133, 172, 226, 1);
+
+  display: block;
+  border: 1px solid rgba(0, 0, 0, 0.5);
+  display: flex;
+  position: relative;
+  border-radius: 3px;
+  min-height: $height;
+  font-family: Helvetica, Arial, sans-serif;
+
+  &--has-error {
+    border: 1px solid rgba(219, 0, 0, 0.5);
+  }
+
+  &__input {
```

Comment:
We haven't really discussed this yet but are we going to use shadow dom? If so, I can easily switch up the notation.
If we decide not to use the shadow dom yet, I'd like to introduce the BEM notation to avoid conflicts in css components

Reply:
You use the Shadow DOM already in this component. Shadow DOM is one the main benefits of Web Components (encapsulation), so I don't see a reason why not. And even if you don't use Shadow DOM, Stencil provides styled CSS support out of the box (`styled: true`). So BEM notation is not needed either way.